docs/core/project-sdk/msbuild-props.md

```diff
@@ -443,6 +443,36 @@ Restoring a referenced package installs all of its direct dependencies and all t
 </PropertyGroup>
 ```
 
+## Run properties
+
+The following properties are used for launching an app during development in Visual Studio or with the [`dotnet run`](../tools/dotnet-run.md) command:
```

Comment:
I don't think most of these properties are used in VS.  VS uses launchSettings.json for this type of information.